### DISCUSSION TITLE
Xenium - "Barcode file missing"

### DISCUSSION THREAD

**Discussion by amoore-vm** [satijalab/seurat#9570] [Q&A] [answered]
I'm exploring the public Xenium datasets, but none contain the expected barcodes.tsv.gz file:

`> library(Seurat)
> library(SeuratObject)
> 
> xenium.obj <- LoadXenium("../Xenium_V1_FFPE_Human_Brain_Alzheimers_With_Addon_outs/", fov = "fov")`
Error in Read10X(data.dir = file.path(data.dir, "cell_feature_matrix/")) : 
  Barcode file missing. Expecting barcodes.tsv.gz

How do you load Xenium data?

Public datasets available here: https://www.10xgenomics.com/datasets?query=Xenium
I've tried human brain Alzheimers, human brain healthy, human breast IDC and mouse brain coronal, but the structure is the same for each: cell_feature_matrix does not contain barcodes.tsv.gz

**Comment by rharao** [comment, marked as answer]
@amoore-vm Thanks for your question. Seurat developers have fixed this, but the fix has not yet made it to the main release version of Seurat. Please see the response at https://github.com/satijalab/seurat/issues/9468 for instructions to install the development version of Seurat.

Also, please do tell us if you run into the same error after installing the development version.

**Comment by amoore-vm** [comment]
Thank you @rharao  That resolved my issue.
